### PR TITLE
Move to new carpet settings manager

### PR DESCRIPTION
**Pull request by adryd325**
TODO:
- [x] Fix CarpetRulePrinter crashing cause translations aren't loaded early enough
- [x] Figure out if validators added by CarpetRulePrinter are still necesary (they're not)

Other notable changes:
- Update carpet version to 1.4.83
- Update minecraft version and mappings to 1.19.1
- Bumps carpet fixes version to 1.11.3

### CODE REVIEW

File: src/main/java/carpetfixes/mixins/entityFixes/FallingBlockEntity_portalsMixin.java [hunk 1]
```diff
@@ -27,11 +27,11 @@ public FallingBlockEntity_portalsMixin(EntityType<?> type, World world) {
...
             at = @At("HEAD")
     )
-    public void tickNetherPortal(CallbackInfo ci) {
+    public void tickPortal(CallbackInfo ci) {
         if (CFSettings.fallingBlocksCantUseNetherPortalsFix) {
-            this.tickNetherPortal();
+            this.tickPortal();
         } else if (CFSettings.fallingBlocksCantReuseGatewaysFix) {
-            this.tickNetherPortalCooldown(); // Improperly named, it's used for gateways too
+            this.tickPortal(); // Improperly named, it's used for gateways too
```
Comment (FxMorin) [inline src/main/java/carpetfixes/mixins/entityFixes/FallingBlockEntity_portalsMixin.java]: You can remove that comment now that the naming makes sense

File: src/main/java/carpetfixes/settings/CustomSettingsManager.java [hunk 2]
```diff
@@ -91,37 +94,23 @@ public int printAllRulesToLog(String category) {
...
-                    validators.add(callConstructor(Validator._COMMAND_LEVEL_VALIDATOR.class));
-                }
-            }
-            if (isStrict && !options.isEmpty()) {
-                if (type == boolean.class || type == int.class || type == double.class || type == float.class) {
-                    validators.add(callConstructor(Validator._STRICT_IGNORECASE.class));
-                } else {
-                    validators.add(callConstructor(Validator._STRICT.class));
-                }
-            }
```
Comment (FxMorin) [inline src/main/java/carpetfixes/settings/CustomSettingsManager.java:109]: Was this removed because its handled by carpet now?

**Review by FxMorin — CHANGES_REQUESTED** [review]

File: src/main/java/carpetfixes/settings/CustomSettingsManager.java (hunk 2 above)
Comment (adryd325) [reply src/main/java/carpetfixes/settings/CustomSettingsManager.java:109]: It looks to have been removed from carpet
I can readd if you'd like, It just doesnt look necesary to generate the wiki
or at least I couldnt figure out what it did

**Comment by adryd325** [comment]
did some super quick sanity tests to make sure i didnt break anthing egregious and this looks ready to merge

**Pull request merged**